CONTRIBUTING.md

```diff
@@ -78,6 +78,52 @@ make lint
 
 Note: The lint tasks require the [codespell](https://github.com/codespell-project/codespell) application. Be sure to install version 2.0 or newer before running the lint tasks.
 
+### Local Development/Testing
+
+It's very helpful to be able to test out changes to Contour locally without building images and pushing into clusters.
+
+To accomplish this, Envoy can be run inside a Kubernetes cluster, typically a `kind` cluster.
+Then Contour is run on your local machine and Envoy will be configured to look for Contour running on your machine vs running in the cluster.
+
+1. Create a kind cluster
+
+```shell
+kind create cluster --config=./examples/kind/kind-expose-port.yaml --name=contour
+```
+
+2. Deploy Contour & Deps to cluster:
```

Comment:
Not essential, but maybe worth noting that the Contour deployment/service/whatever else can be deleted from the cluster because they won't be used. Helps avoid some confusion.